test/command/suite/normalizers/table/checks.expected

```diff
@@ -0,0 +1,37 @@
+table_create Substitutions TABLE_PAT_KEY ShortText
+[[0,0.0,0.0],true]
+column_create Substitutions substituted COLUMN_SCALAR ShortText
+[[0,0.0,0.0],true]
+load --table Substitutions
+[
+{"_key": "あ", "substituted": "<あ>"},
+{"_key": "いち", "substituted": "1"}
+]
+[[0,0.0,0.0],2]
+normalize   'NormalizerTable("column", "Substitutions.substituted")'   ".あ。いち."   WITH_CHECKS
+[
+  [
+    0,
+    0.0,
+    0.0
+  ],
+  {
+    "normalized": ".<あ>。1.",
+    "types": [
+
+    ],
+    "checks": [
```

Comment:
What does `checks` represent?
I couldn't fully understand it from the code...

Reply:
It has the number of normalized bytes elements. 1(.) + 1(<) + 3(あ) + 1(>) + 3(。) + 1(1) + 1(.) = 11 in this case.

Each element represents the source character size in bytes.

If there are more normalized characters than source characters, elements of the rest characters are `-1`. `あ` is normalized to `<あ>` in this case. `checks[2]` is for (normalized) `あ` and `checks[5]` is for `>`. They are `-1` because the source character is only `あ`.

If normalized character is a multibyte character, the element of the first byte has the source character size in bytes. The rest elements are `0`. `checks[2]`, `checks[3]` and `checks[4]` are for (normalized) `あ`. `checks[3]` and `checks[4]` are `-1` because they are not for the first byte.

Reply:
I see, Thank you!

> `checks[3]` and `checks[4]` are `-1` because they are not for the first byte.
`-1` is just typo of `0`, right?

Reply:
Ah, right.